src/test/java/core/basesyntax/service/FruitShopServiceImplTest.java

```diff
@@ -0,0 +1,73 @@
+package core.basesyntax.service;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+import core.basesyntax.model.Storage;
+import core.basesyntax.model.TransactionDto;
+import core.basesyntax.service.operations.AdditionHendlerImpl;
+import core.basesyntax.service.operations.OperationHendler;
+import core.basesyntax.service.operations.SubstractionHendlerImpl;
+import core.basesyntax.service.strategy.OperationsStrategy;
+import core.basesyntax.service.strategy.OperationsStrategyImpl;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitShopServiceImplTest {
+    private static final Map<TransactionDto.OperationTypes, OperationHendler> strategy =
+            new HashMap<>();
+    private static final OperationsStrategy operationsStrategy =
+            new OperationsStrategyImpl(strategy);
+    private static final Map<String, Integer> FRUIT_COUNT = Storage.FRUIT_COUNT;

```

Comment:
the same thing - an extra variable. You can use direct access to Storage.FRUIT_COUNT